beacon_node/network/src/sync/block_lookups/mod.rs

```diff
@@ -1368,6 +1319,36 @@ impl<T: BeaconChainTypes> BlockLookups<T> {
         self.parent_lookups.drain(..).len()
     }
 
+    pub fn downscore_on_rpc_error(
+        &self,
+        peer_id: &PeerId,
+        error: &RPCError,
+        cx: &SyncNetworkContext<T>,
+    ) {
+        // Note: logging the report event here with the full error display. The log inside
+        // `report_peer` only includes a smaller string, like "invalid_data"
+        debug!(self.log, "reporting peer for sync lookup error"; "error" => %error);
+        if let Some(action) = match error {
+            // Protocol errors are heavily penalized
+            RPCError::SSZDecodeError(..)
```

Comment:
Looks like we already downscore here on RPC errors: https://github.com/sigp/lighthouse/blob/unstable/beacon_node/lighthouse_network/src/service/mod.rs#L1407

So on everything apart from the new `InvalidData`s we'd double downscore.   My suggestion would be to revive the `LookupVerifyError` and add a wrapper for the two:
```
enum LookupFailure {
    LookupVerifyError(LookupVerifyError),
    RpcError(RpcError)
}

match e {
    LookupVerifyError(e) => {..} // downscore and drop,
    RpcError(e) => {..} // drop
}
```
Or maybe add an `RpcError` variant to the `LookupVerifyError`.

Reply:
Made these updates here: https://github.com/dapplion/lighthouse/pull/24/commits/7ed104f23a5fa4462343613e0b00af91c98ce576